versions/3.0.4.md

```diff
@@ -176,32 +176,37 @@ Note that no aspect of implicit connection resolution changes how [URLs are reso
 
 ### Data Types
 
-Data types in the OAS are based on the types supported by the [JSON Schema Specification Wright Draft 00](https://tools.ietf.org/html/draft-wright-json-schema-00#section-4.2).
-Note that `integer` as a type is also supported and is defined as a JSON number without a fraction or exponent part.
-`null` is not supported as a type (see [`nullable`](#schema-nullable) for an alternative solution).
-Models are defined using the [Schema Object](#schema-object), which is an extended subset of JSON Schema Specification Wright Draft 00.
+Data types in the OAS are based on the non-`null` types supported by the [JSON Schema Validation Specification Draft Wright-00](https://datatracker.ietf.org/doc/html/draft-wright-json-schema-validation-00#autoid-32):
+"boolean", "object", "array", "number", "string", or "integer".
+See [`nullable`](#schema-nullable) for an alternative solution to "null" as a type.
```

Comment:
Oh wow! I totally forgot about "null" when we were working on the 3.0.4 version. We need to exclude null there too, right?

Reply:
@mikekistler this is the 3.0.4 version- I did this change backwards (3.1.1 first) because 3.1.1 was easier as it uses JSON Schema as-is.  Hopefully I didn't miss anything in this PR where `null` needed to be removed.